vault/token_store.go

```diff
@@ -602,26 +586,9 @@ type tsRoleEntry struct {
 	// If true, tokens created using this role will be orphans
 	Orphan bool `json:"orphan" mapstructure:"orphan" structs:"orphan"`
 
-	// If non-zero, tokens created using this role will be able to be renewed
-	// forever, but will have a fixed renewal period of this value
-	Period time.Duration `json:"period" mapstructure:"period" structs:"period"`
```

Comment:
Should we be deprecating these instead of removing them, and also have `tokenStoreRole` be able to handle upgrades?

Reply:
They're not deprecated or removed! They're now a part of the embedded TokenParams struct, with the same name/type/JSON name.